<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QWidget" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>300</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton
{
	margin:0;
	padding:0;
	border: 1px solid;
	border-radius: 6px;
	background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1, fx:0.25, fy:0.273, stop:0.0738636 rgba(192, 192, 192, 255), stop:0.357955 rgba(129, 129, 129, 255), stop:0.903409 rgba(0, 0, 0, 0));
}
QToolButton:pressed {
    background-color: lightgray;
}
QToolButton:pressed:checked {
    background-color: lightgray;
}
QToolButton:checked {
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:1, fx:0.25, fy:0.273, stop:0.0738636 rgba(255, 255, 255, 255), stop:0.352273 rgba(255, 255, 127, 234), stop:0.903409 rgba(0, 0, 0, 0));
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTextBrowser" name="tbText">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextBrowser {
	background-color: black;
	color: yellow;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
     <property name="openLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="tbPublic">
         <property name="toolTip">
          <string>Public (nearby)</string>
         </property>
         <property name="text">
          <string>A</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbWorld">
         <property name="toolTip">
          <string>World</string>
         </property>
         <property name="text">
          <string>W</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbClan">
         <property name="toolTip">
          <string>Clan</string>
         </property>
         <property name="text">
          <string>C</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbGroup">
         <property name="toolTip">
          <string>Group</string>
         </property>
         <property name="text">
          <string>G</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbPrivate">
         <property name="toolTip">
          <string>Private</string>
         </property>
         <property name="text">
          <string>P</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="cbType">
       <property name="toolTip">
        <string>Target chat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leInput">
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="maxLength">
        <number>80</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>leInput</tabstop>
  <tabstop>cbType</tabstop>
  <tabstop>tbText</tabstop>
 </tabstops>
 <resources>
  <include location="../smile/smile.qrc"/>
 </resources>
 <connections/>
</ui>
